### PR TITLE
devopsboard.com has been added into management systems

### PR DESCRIPTION
**Pull request by evgeny-savitsky**
Here is another usefull tool we use and suggest others to use it in their daily work

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -146,6 +146,7 @@ Table of Contents
 
   * https://bitnami.com/ - Deploy prepared apps on IaaS. Management of 1 AWS micro instance free
   * http://www.visualops.io/ - 3,600 instance hours per month free
+  * http://devopsboard.com/ - Manage software and hardware issues aggregated from monitoring tools on single Kanban board
```
Comment (ripienaar) [inline README.md]: does this have a free tier or is it just a trial? looks like just a trial to me which wouldnt fit the goal of this list

Comment (evgeny-savitsky) [inline README.md]: Can't say more clear, we use it free

Comment (evgeny-savitsky) [inline README.md]: Actually targetprocess, jira and so on are not free at all too

Comment (ripienaar) [inline README.md]: All I see on their pricing page regarding it is:

```Submit the credentials on the left and get your instance ready in couple of seconds. We give you 14-days trial period to use DevOps board absolutely free.```

which suggests its a trial only

Comment (evgeny-savitsky) [inline README.md]: there is hosted option, maybe they're providing it free for open-source or academic needs?

Comment (ripienaar) [inline README.md]: does not seem that way, you added it here - based on where did you see its free? if I cant see its free I cant add it

Comment (evgeny-savitsky) [inline README.md]: Ok, sorry for that

**Pull request closed without merging**